gnucash/import-export/import-backend.h

```diff
@@ -64,36 +64,22 @@ typedef enum _action
  *
  * @param trans The transaction for which to check for an existing
  * online_id. */
-gboolean gnc_import_exists_online_id (Transaction *trans);
+gboolean gnc_import_exists_online_id (Transaction *trans, GHashTable* Foo);
```

Comment:
Let's not use `Foo` and besides the [coding standards](https://wiki.gnucash.org/wiki/CodingStandard) say that capitalized and camel case identifiers should be type names. Variables should be lower-case and joined with underscores.

Reply:
Yep, apologies, definitely slipped through the cracks.